model/jpa/src/main/resources/META-INF/jpa-changelog-26.0.0.xml

```diff
@@ -28,4 +28,22 @@
         <addUniqueConstraint tableName="ORG" columnNames="REALM_ID, ALIAS" constraintName="UK_ORG_ALIAS"/>
     </changeSet>
 
+    <changeSet author="keycloak" id="26.0.0-org-group">
+        <addColumn tableName="KEYCLOAK_GROUP">
+            <column name="TYPE" type="INT" defaultValueNumeric="0"/>
+        </addColumn>
+        <update tableName="KEYCLOAK_GROUP">
+            <column name="TYPE" valueNumeric="0"/>
+        </update>
+        <update tableName="KEYCLOAK_GROUP">
+            <column name="TYPE" valueNumeric="1" type="INT"/>
+            <where>NAME IN (SELECT ID FROM ORG)</where>
```

Comment:
Not sure do you have a better suggestion?

Reply:
I've added back a `customChange` to get the right schema. But still, the migration step does not work because of https://github.com/keycloak/keycloak/actions/runs/9678735849/job/26703371471?pr=30590#step:4:2119.

I'm not sure if explicitly setting collations is something we want or if there is another way of doing the same but without going into the same problem?